pkg/gopass/secrets/akv.go

```diff
@@ -310,3 +330,32 @@ func (a *AKV) FromMime() bool {
 func (a *AKV) SafeStr() string {
 	return "(elided)"
 }
+
+// pwWriter is a io.Writer that will extract the first line of the input stream and
+// then write it to the password field of the provided AKV. The first line can stretch
+// multiple chunks but once the first line has been completed any writes to this
+// writer will be silently discarded.
+type pwWriter struct {
+	a       *AKV
+	buf     strings.Builder
+	written bool
+}
+
+func (p *pwWriter) Write(buf []byte) (int, error) {
+	n := len(buf)
+	if p.written || p.a == nil {
+		return n, nil
```

Comment:
```suggestion
		return 0, nil
```
Since no writes are done in that case, no?

Reply:
Actually this did raise some questions with me that I didn't have a good answer for and I ditched the MultiWriter approach for a pass-through wirter. I think that's easier to reason about.